src/test/sh/Plugin.Testing.Stub-1.0.0/usr/opt/EXAplugins/Testing.Stub-1.0.0/exaoperation-gate/plugin-functions

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
```

Comment:
Please use Bash instead of python. We want to keep the number of programming languages per project to an absolute minimum.

Reply:
As discussed, I'll leave that one for you :)

Reply:
Roger and wilco.